src/main/webapp/app/exercises/programming/manage/update/update-components/programming-exercise-language.component.ts

```diff
@@ -12,6 +12,7 @@ export class ProgrammingExerciseLanguageComponent {
     readonly ProjectType = ProjectType;
 
     @Input() isImport: boolean;
+    @Input() isImportFromFile: boolean;
```

Comment:
is isImport also true when isImportFromFile is true?
This might lead to confusion

Reply:
no, these are two separate things.
But I get your point, any suggestions how to improve the naming.?

Reply:
Renaming isImport to isImportFromExistingExercise or something like that